tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AWS/DataStreamsMonitoringAwsSqsTests.cs

```diff
@@ -29,17 +29,33 @@ public DataStreamsMonitoringAwsSqsTests(ITestOutputHelper output)
 
     public static IEnumerable<object[]> GetEnabledConfig()
     {
-        return from packageVersionArray in PackageVersions.AwsSqs
-               select new[] { packageVersionArray[0] };
+        foreach (var packageVersionArray in PackageVersions.AwsSqs)
+        {
+            foreach (var batch in new[] { 0, 1 })
+            {
+                foreach (var inThread in new[] { 0, 1 })
+                {
+                    foreach (var inject in new[] { 0, 1 })
+                    {
+                        yield return [packageVersionArray[0], batch, inThread, inject];
+                    }
+                }
+            }
+        }
```

Comment:
nit, matter of choice, but I'm biased 😄 
```suggestion
        return from batch in new[] { 0, 1 }
               from inThread in new[] { 0, 1 }
               from inject in new[] { 0, 1 }
               from packageVersionArray in PackageVersions.AwsSqs
               select [packageVersionArray[0], batch, inThread, inject];
```